src/components/containers/branding.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactNode } from 'react'
+import { DerivStore } from 'store'
+
+type BrandingProps = {
+    children: ReactNode
+}
+
+const Branding = ({ children }: BrandingProps) => {
+    const { is_show_branding } = React.useContext(DerivStore)
+    return is_show_branding && <>{children}</>
```

Comment:
If `is_show_branding` is `false` what this component will return?

Reply:
If it is true,it will hide the children components..if not,it will show the component

Reply:
Don't we need `return null` for `false` condition?

Reply:
okay i am returning null for false condition